gui/setup.iss

```diff
@@ -39,27 +39,27 @@ Name: "desktopicon"; Description: "{cm:CreateDesktopIcon}"; GroupDescription: "{
 [Files]
 ; This isn't as wreckless as it seems; these directories are populated on a
 ; controlled way by the batch file.
-Source: qtdir\bin\*.dll;		DestDir: "{app}"; Flags: ignoreversion
-Source: qtdir\plugins\*; 		DestDir: "{app}\plugins"; Flags: ignoreversion recursesubdirs createallsubdirs
+;Source: qtdir\bin\*.dll;		DestDir: "{app}"; Flags: ignoreversion

```

Comment:
Is the whole block of using qtdir dead and replaced by the copying below? If so, should we nuke this and teh related comments?

Reply:
agreed.  will nuke.  additionally mksetup.bat is obsoleted by this PR and should be deleted from the HEAD.